ckan/controllers/organization.py

```diff
@@ -1,9 +1,32 @@
 # encoding: utf-8
 
+import logging
```

Comment:
Sort alphabetically, please

Reply:
But it's already sorted, L goes before R, and builtin packages separate from written with space.